examples/federation/base-app/src/main/kotlin/com/expediagroup/graphql/examples/federation/base/schema/Widget.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.expediagroup.graphql.examples.model
+package com.expediagroup.graphql.examples.federation.base.schema
```

Comment:
`schema` package implies (at least to me) something related to either loading or modifying the schema, this package holds data model that is exposed in the schema but i guess as long as we use consistent packages then it should be fine